packages/react-composites/src/composites/common/ControlBar/DesktopMoreButton.tsx

```diff
@@ -121,6 +124,22 @@ export const DesktopMoreButton = (props: DesktopMoreButtonProps): JSX.Element =>
       },
       disabled: props.disableButtonsForHoldScreen
     });
+
+    if (props.onCaptionsSettingsClick) {
+      captionsContextualMenuItems.push({
+        key: 'openCaptionsSettingKey',
+        text: localeStrings.strings.call.captionsSettingLabel,
+        onClick: props.onCaptionsSettingsClick,
+        iconProps: {
+          iconName: 'SettingsIcon',
+          styles: { root: { lineHeight: 0 } }
```

Comment:
Is there a concern of lineHeight: 0? Would be nice to add a comment here

Reply:
Just following icon styles for other more buttons